addOns/scripts/src/main/java/org/zaproxy/zap/extension/scripts/automation/ScriptJob.java

```diff
@@ -45,7 +45,7 @@ public class ScriptJob extends AutomationJob {
 
     public static final String JOB_NAME = "script";
     public static final String PARAM_ACTION = "action";
-    public static final String PARAM_SCRIPT_NAME = "scriptName";
+    public static final String PARAM_SCRIPT_NAME = "name";
```

Comment:
IMO this (along with `getCustomConfigParameters()`) should be removed instead, tthey are not custom config parameters, they are normal parameters (see `getParameters()`).